src/plugins/my-kiva-homepage-mixin.js

```diff
@@ -0,0 +1,27 @@
+import myKivaForAllUsersQuery from '#src/graphql/query/shared/myKivaForAllUsers.graphql';
+import { getIsMyKivaEnabled, MY_KIVA_FOR_ALL_USERS_KEY } from '#src/util/myKivaUtils';
+import { readBoolSetting } from '#src/util/settingsUtils';
+
+export default {
+	data() {
+		return {
+			redirectToMyKivaHomepage: false,
+		};
+	},
+	created() {
+		const myKivaData = this.apollo.readQuery({
+			query: myKivaForAllUsersQuery,
+		});
+
+		const userData = myKivaData?.my ?? null;
+		const myKivaFlagEnabled = readBoolSetting(myKivaData, MY_KIVA_FOR_ALL_USERS_KEY);
+
+		this.redirectToMyKivaHomepage = getIsMyKivaEnabled(
```

Comment:
We only want to redirect the user if they are logged in, since visitors can't see MyKiva.